README.md

```diff
@@ -1,2 +1,167 @@
-# Full-Stack-Capstone-Backend
-Explore the backend of Doctor Reservation App! This repository, powered by Rails, manages user authentication, reservation logic, and API endpoints.
+# Full Stack Capstone Backend<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [Full Stack Capstone Backend](#tdd-project)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 TDD-Project ](#-tdd-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Run tests](#run-tests)
+    - [Run console](#run-console)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Final Capstone <a name="about-project"></a>
+
+**Full Stack Capstone Backend** Explore the backend of Car Reservation App! This repository, powered by Rails, manages user authentication, reservation logic, and API endpoints.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+   <li>Rails</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"><a>
+
+- [Greeting-page]
+
+## Link to the Frontend Repo
+
+- [Frontend](https://github.com/PabloEGonz/full-stack-capstone-front-end)
+
+<!-- Getting Started -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To run the project locally, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to:
+
+- Install [Git](https://git-scm.com/)
+- Install [Ruby](https://www.ruby-lang.org/en/)
+- Install a code editor, I suggest [VsCode](https://code.visualstudio.com/)
+
+### Setup
+
+Clone the repo into your local machine:
+
+```bash
+git clone https://github.com/ezraermy/Full-Stack-Capstone-Backend.git
+```
+
+Install the dependencies using npm:
+
+```bash
+bundler install
+```
+
+
+
+### Run tests
```

Comment:
- [ ] You have some functional model tests, but there are no instructions about how to run them.

Kindly, include a "tests" section on your readme with specific instructions about how to run your models tests:

![image](https://github.com/ezraermy/Full-Stack-Capstone-Backend/assets/57421823/a0b37941-0146-413d-9e2e-80ab0c5eccdf)